src/Microsoft.Identity.Web/MicrosoftIdentityOptions.cs

```diff
@@ -70,25 +70,25 @@ public class MicrosoftIdentityOptions : OpenIdConnectOptions
         /// <summary>
         /// Gets or sets the edit profile user flow name for B2C, e.g. b2c_1_edit_profile.
         /// </summary>
-        public string EditProfilePolicyId { get; set; }
+        public string? EditProfilePolicyId { get; set; }
 
         /// <summary>
         /// Gets or sets the sign up or sign in user flow name for B2C, e.g. b2c_1_susi.
         /// </summary>
-        public string SignUpSignInPolicyId { get; set; }
+        public string? SignUpSignInPolicyId { get; set; }
 
         /// <summary>
         /// Gets or sets the reset password user flow name for B2C, e.g. B2C_1_password_reset.
         /// </summary>
-        public string ResetPasswordPolicyId { get; set; }
+        public string? ResetPasswordPolicyId { get; set; }
 
         /// <summary>
         /// Gets the default user flow (which is signUpsignIn).
         /// </summary>
-        public string DefaultUserFlow => SignUpSignInPolicyId;
+        public string? DefaultUserFlow => SignUpSignInPolicyId;
 
         /// <summary>
-        /// Is considered B2C if the attribute SignUpSignInPolicyId is defined.
+        /// Gets a value indicating whether app is considered B2C if the attribute SignUpSignInPolicyId is defined.
```

Comment:
nit: the previous comment is clearer imho

Reply:
Yea, I agree. Reverted it. I'm not a huge fan of that analyzer suggestion myself.

Reply:
The analyzer suggests that?